### DISCUSSION TITLE
Is the "group_scences(a0)" method not available?

### DISCUSSION THREAD

**Discussion by lucas-ita** [cwapi3d/cwapi3dpython#164] [Q&A] [answered]
I realize that in the CW API there is the `group_scences(a0)` method but when I try to use it in the console it seems that it is not available.

![image](https://github.com/cwapi3d/cwapi3dpython/assets/123772761/68bf5a0a-cbab-4ee9-8bdf-c33c25ef323a)

**Comment by Brunner246** [comment, marked as answer]
Hei,
You have to update cadwork 3d.

```
 cadwork 3d (SP2024)
============
 Build ........................: 30.0.454
 Date .........................: 2024-03-26
 ```

**Reply by lucas-ita** [reply]
![image](https://github.com/cwapi3d/cwapi3dpython/assets/123772761/52556698-02e6-4e67-b334-149de3a606e6)
![image](https://github.com/cwapi3d/cwapi3dpython/assets/123772761/65d2cfe4-d9b9-456a-92f1-b0f3de027640)

**Reply by Brunner246** [reply]
U need the SP2024 version, please contact ur support office.
